templates/expo-template-bare-minimum/ios/HelloWorld/AppDelegate.m

```diff
@@ -63,14 +63,17 @@ - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge {
 
 // Linking API
 - (BOOL)application:(UIApplication *)application openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options {
-  return [RCTLinkingManager application:application openURL:url options:options];
+  return [super application:application openURL:url options:options] || [RCTLinkingManager application:application openURL:url options:options];
```

Comment:
```suggestion
  [super application:application openURL:url options:options];
  return [RCTLinkingManager application:application openURL:url options:options];
```

Reply:
@Kudo, why? If the subscriber handled the request (returned `true`), RCTLinkingManager shouldn't be called I think 🤔

Reply:
to be honest, i have no clear answer of either one to be right.
if a handler returns true there, will other modules notice this behavior or developer notice that `Linking` will break?

maybe in the next sdk we should either (cc @brentvatne who mentioned this initially):
- visit all subscriber interfaces and clearly to mention returning `true` with caution.
- logging for which handlers we walk through and which one win the final competition (in the case winner will take over others default behavior, this is especially for `reactDelegate.createBridge`)

Reply:
My thinking was to try copying the [legacy wrapper](https://github.com/expo/expo/blob/35768a6f562d451d1fd4db4e025daf0a008aaf35/packages/expo-modules-core/ios/AppDelegates/EXLegacyAppDelegateWrapper.m#L61).